mongomock/helpers.py

```diff
@@ -15,7 +15,7 @@
 try:
     from bson import RE_TYPE
 except ImportError:
-    if sys.version_info[0] > 2 and sys.version_info[1] > 6:
+    if sys.version_info > (3, 7):
```

Comment:
@helloqiu I think you meant `>=`

Reply:
![image](https://user-images.githubusercontent.com/8971659/43043278-8fde1efc-8dc2-11e8-9ce9-e79057ae7955.png)
I think it should be `>`.